.github/workflows/build-solution.yml

```diff
@@ -17,15 +17,14 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
+    - name: Check out repository
+      uses: actions/checkout@v3
       with:
         fetch-depth: 0
         submodules: recursive
 
-    - name: Setup .NET
-      uses: actions/setup-dotnet@v2
-      with:
-        dotnet-version: 6.0.x
```

Comment:
Shouldn't this be changed to 7 instead of removing it completely? 
Or you can use [global-json-file](https://github.com/actions/setup-dotnet#using-the-global-json-file-input) instead of a version.

Reply:
Yes, it uses `global.json` instead.

Reply:
Is that default behavior?

Reply:
Yep, the default location is repo root.

Reply:
I can't see it on their documentation though, are you sure you're not supposed to add: _global-json-file_

Reply:
[Here](https://learn.microsoft.com/en-us/dotnet/core/tools/global-json) you go